packages/client/src/redux/TrecipeList/types.ts

```diff
@@ -15,11 +15,34 @@ export const initialState: TrecipeListState = {
     myAssociatedTrecipes: [],
 };
 
-export enum TrecipeListActionTypes {
-    ADD_TRECIPE = '@trecipeList/ADD_TRECIPE',
+export enum TrecipeListActionCategory {
+    CREATE_TRECIPE = '@trecipeList/CREATE_TRECIPE',
     DELETE_TRECIPE = '@trecipeList/DELETE_TRECIPE',
     UPDATE_TRECIPE = '@trecipeList/UPDATE_TRECIPE',
-    LOAD_TRECIPE = '@trecipeList/LOAD_TRECIPE',
-    LOAD_ASSOCIATED_TRECIPE = '@trecipeList/LOAD_ASSOCIATED_TRECIPE',
-    LOAD_MY_ASSOCIATED_TRECIPE = '@trecipeList/LOAD_MY_ASSOCIATED_TRECIPE',
+    FETCH_MY_TRECIPES = '@trecipeList/FETCH_MY_TRECIPES',
+    FETCH_ASSOCIATED_TRECIPES = '@trecipeList/FETCH_ASSOCIATED_TRECIPES',
+    FETCH_MY_ASSOCIATED_TRECIPES = '@trecipeList/FETCH_MY_ASSOCIATED_TRECIPES',
+}
+
+export enum TrecipeListActionTypes {
```

Comment:
Whole bunch of action types. Basically, each action needs a set of `SUCCESS`, `FAILURE` and `REQUEST`. Not the cleanest method... but hey, it works :P